main/res/drawable/back.xml

```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportHeight="24"
+    android:viewportWidth="24"
+    android:tint="@color/colorIcon"
```

Comment:
Not sure, but would it make sense to put those size infos into dimens.xml for having (and keeping) the sizes consistent? (And to make our lives easier if we ever come to a point where we might want to change that size.)

Reply:
Probably I would wait until we change the size, because that would mean for manual editing when adding new icons

Reply:
I'm with @murggel. It makes IMHO absolutely no sense to extract the sizes as this is **not** the real display size but just part of the SVG code. AFAIK changing that size will even distort the icon.

The real display size is contained somewhere in the material theme / our style definitions...

Reply:
Ah, I see, thanks for explaining. Sorry, I've close to no idea of the SVG format, just saw the repeated size definitions.

Reply:
@fm-sys answer is correct, height and width does not really change the size of the displayed icon, I tried that for the info and compass icon to make them smaller, and it did not work.
The viewport-dimensions are the size of the drawn icon, so changing that would crop the image.